j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/ClosureTask.java

```diff
@@ -176,7 +178,20 @@ public Task resolve(Project project, Config config) {
         CompilerOptions.LanguageMode languageOut = CompilerOptions.LanguageMode.fromString(config.getLanguageOut());
         //TODO probably kill this, or at least make it work like an import via another task so we detect changes
         Collection<String> externs = config.getExterns();
-        Optional<File> translationsfile = config.getTranslationsFile();
+
+        TranslationsFileProcessor translationsFileProcessor = new TranslationsFileProcessor(config);
+        translationsFileProcessor.setProjectInputs(
```

Comment:
i don't like it, but looks like it's the only place i can pass .xtb to processor
@niloc132 could we do better ?

Reply:
@niloc132 as you asked me, i created an issue to track it https://github.com/Vertispan/j2clmavenplugin/issues/177